app/build.gradle

```
@@ -31,16 +32,18 @@ android {
 }
 
 dependencies {
-    compile fileTree(include: ['*.jar'], dir: 'libs')
-    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    androidTestImplementation ('androidx.test.espresso:espresso-core:3.6.1', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    testCompile 'junit:junit:4.12'
-    compile 'org.literacyapp:literacyapp-model:1.1.53'
-    compile 'org.literacyapp:contentprovider:1.0.9'
-    compile 'org.tensorflow:tensorflow-android:1.2.0-rc0'
-    compile 'org.greenrobot:greendao:3.2.0'
-    compile 'com.android.support:appcompat-v7:23.4.0'
-    compile 'com.android.support:cardview-v7:23.4.0'
-    compile 'com.android.support:support-v4:23.4.0'
+    testImplementation 'junit:junit:4.13.2'
+    implementation 'androidx.appcompat:appcompat:1.7.0'
+    implementation 'androidx.cardview:cardview:1.0.0'
+    implementation 'androidx.core:core-ktx:1.16.0'
+    implementation 'ai.elimu:content-provider:1.3.23@aar'
```

@saedmanaf Try removing `.aar` so that we get a similar setup as the one used at https://github.com/elimu-ai/sound-cards/commit/f9d3928cbd19c586f65a556b9e157b6e4e4af700
```suggestion
    implementation 'ai.elimu:content-provider:1.3.23'
```